webpack/assets/javascripts/react_app/components/common/AutoComplete/AutoComplete.test.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { render } from 'enzyme';
+import toJson from 'enzyme-to-json';
+
+import AutoComplete from './index';
+import { countItems } from './AutoComplete.fixtures';
+
+describe('Autocomplete component', () => {
+  it('should render input but not items on initial render', () => {
+    const wrapper = render(<AutoComplete
+        items={countItems}
+        onInputUpdate={() => {}}
+        onSelect={() => {}}
+      />);
+    expect(toJson(wrapper)).toMatchSnapshot();
+  });
```

Comment:
Does it full coverage ?

Reply:
@danseethaler you can create a manual mock inside the `__mocks__` folder:
https://facebook.github.io/jest/docs/en/manual-mocks.html

Reply:
@sharvit thanks for the idea. That got me thinking and I ended up abstracting the rendering of items so I could test it independently. Now we've got full coverage on the component. Thanks!